README.md

```diff
@@ -1,160 +1,12 @@
-
-
-
-### 20. How to Build a Chatbot with the Chat Endpoint
-Learn how to build a chatbot using Cohere’s Chat endpoint.
-[<a href="https://github.com/cohere-ai/notebooks/blob/main/notebooks/How_to_Build_a_Chatbot_with_the_Chat_Endpoint.ipynb
-">Notebook</a> | <a href="https://colab.research.google.com/github/cohere-ai/notebooks/blob/main/notebooks/How_to_Build_a_Chatbot_with_the_Chat_Endpoint.ipynb
-">Colab</a>]
-
-<img src="https://raw.githubusercontent.com/cohere-ai/notebooks/main/notebooks/images/chatbot-chat-endpoint.png" style="width:100%; max-width:400px" alt="How to Build a Chatbot with the Chat Endpoint" />
-
-### 21. How to Build a RAG-Powered Chatbot with Chat, Embed, and Rerank
-Learn how to build a chatbot using Cohere’s Chat, Embed, and Rerank endpoints.
-[<a href="https://github.com/cohere-ai/notebooks/blob/main/notebooks/RAG_Chatbot_with_Chat_Embed_Rerank.ipynb
-">Notebook</a> | <a href="https://colab.research.google.com/github/cohere-ai/notebooks/blob/main/notebooks/RAG_Chatbot_with_Chat_Embed_Rerank.ipynb
-">Colab</a>]
-
-<img src="https://raw.githubusercontent.com/cohere-ai/notebooks/main/notebooks/images/rag-chatbot.png" style="width:100%; max-width:400px" alt="How to Build a RAG-Powered Chatbot with Chat, Embed, and Rerank" />
-
-### 22. How to Build a RAG-Powered Chatbot with Connectors
-Learn how to build a chatbot connectors using Cohere's build-your-own connectors framework.
-[<a href="https://github.com/cohere-ai/notebooks/blob/main/examples/chat_rag_connector/RAG_Chatbot_with_Connectors.ipynb
-">Notebook</a> | <a href="https://colab.research.google.com/github/cohere-ai/notebooks/blob/main/examples/chat_rag_connector/RAG_Chatbot_with_Connectors.ipynb
-">Colab</a>]
-
-<img src="https://raw.githubusercontent.com/cohere-ai/notebooks/main/notebooks/images/rag-chatbot-connector.png" style="width:100%; max-width:400px" alt="How to Build a RAG-Powered Chatbot with Connectors" />
-
-### 23. How to Fine-Tune a Model for Cohere Chat
-[<a href="https://github.com/cohere-ai/notebooks/blob/main/notebooks/Fine_tuning_Models_for_the_Chat_Endpoint.ipynb
-">Notebook</a> | <a href="https://colab.research.google.com/github/cohere-ai/notebooks/blob/main/notebooks/Fine_tuning_Models_for_the_Chat_Endpoint.ipynb
-">Colab</a>]
-
-<img src="https://raw.githubusercontent.com/cohere-ai/notebooks/main/notebooks/images/chat-finetune.png" style="width:100%; max-width:400px" alt="How to Fine-Tune a Model for Cohere Chat" />
\ No newline at end of file
+| Category | Description |
+| --- | --- |
+| [LLM University](notebooks/llmu/readme.md) | Guides for getting started with Cohere, starting with basic usage and progressing to advanced topics. The code companion to the full [LLM University course](https://llm.university/).|
```

Comment:
I think this link should be to notebooks/llmu/ and not to notebooks/llmu/readme.md. That way readers can see the list of notebooks in addition to the contents of the readme file.